### DISCUSSION TITLE
What is [system-distro-env] on this doc excerpt?

### DISCUSSION THREAD

**Discussion by wkrueger** [microsoft/wslg#781] [Q&A] [answered]
https://github.com/microsoft/wslg/wiki/WSLg-Configuration-Options-for-Debugging

What whould I replace `[system-distro-env]` with on my machine. This does not look clear.

My wsl --status

```
Default Distribution: Ubuntu-20.04
Default Version: 2
WSL version: 0.61.8.0
Kernel version: 5.10.102.1
WSLg version: 1.0.39
MSRDC version: 1.2.3213
Direct3D version: 1.601.0
DXCore version: 10.0.25131.1002-220531-1700.rs-onecore-base2-hyp
Windows version: 10.0.22000.739
```

I want to enable the WESTON_RDPRAIL_SHELL_ALLOW_ZAP variable because my RDP sometimes freezes. (It already is set up but it seldom works.)

**Comment by Biswa96** [comment, marked as answer]
The .wslgconfig file follows [INI file](https://en.wikipedia.org/wiki/INI_file) syntax. The `[system-distro-env]` is the name of the section. That section name should not be changed.

**Reply by wkrueger** [reply]
ah I thought I maybe had to replace it with [Ubuntu] or sorts, misunderstood.

My WSLg windows frequently freeze. CTRL SHIFT backspace seldom works, so I sometimes have to restart the whole WSL (along with docker and a couple of VS Code windows...)

**Reply by Biswa96** [reply]
BTW, the code is here https://github.com/microsoft/wslg/blob/main/WSLGd/main.cpp. It parses the .wslgconfig file. See `c_systemDistroEnvSection` variable.
